config/services.yaml

```diff
@@ -8,6 +8,17 @@ services:
     _defaults:
         autoconfigure: true
 
+    # controllers are imported separately to make sure services can be injected
+    # as action arguments even if you don't extend any base controller class
+    App\Controller\:
+        resource: '../src/Controller'
+        autowire: true
+        tags: ['controller.service_arguments']
+
+    App\Dashboard\:
```

Comment:
Isn't it resolved out-of-the-box?

Reply:
And just quick thought. I would say, that people are used to having imports at the beginning of the file

Reply:
> Isn't it resolved out-of-the-box?

Not yet, I need to work on that to be closer to Symfony skeleton.

Reply:
> And just quick thought. I would say, that people are used to having imports at the beginning of the file

Yes, I moved these imports on top.